Integrations/integration-CrowdStrikeIntel.yml

```
@@ -333,14 +338,24 @@ script:
             Type: entryTypes.note,
             Contents: res.body,
             ContentsFormat: formats.json,
+            IgnoreAutoExtract: true,
             HumanReadable: md,
             EntryContext: ec
         };
     }
 
     function doIndicator(ind, type, title, appendContextFunc) {
-        var a = {equal: ind};
-        var res = doReq('GET', 'indicator/'+ VERSION +'/search/indicator', a);
+        var a = {'indicator.equal': ind};
+        if (type === 'hash') {
+            var hashType = ind.length === 32 ? 'hash_md5' : ind.length === 40 ? 'hash_sha1' : 'hash_sha256';
+            a['type.equal'] = hashType;
```

maybe its better to return an error if the hash is in bad length.